setup.py

```diff
@@ -1,12 +1,15 @@
+
```

Comment:
Have no way to check what was the error but my linter kept complaining about not leaving a space. I'll check the error and share it here once I get back to my laptop.